### ISSUE TITLE
revise Additional Synonyms 333-326

### ISSUE THREAD

**Issue by arlogriffiths** [erc-dharma/tfd-nusantara-philology#149]
333 Synonyms of Corn
	1 dhānya, sasya, ṅa homan. 
	^4830. dhānya] EdLC J1, dhānya[... J1, lac. J3 (larger gap), lac. J1 (larger gap)
	^4831. dhānya … (335.1) pipakan] A gap due to loss intervenes in J1.

Interesting that (h)oman seems to be unattested in OJED. I don't find any occurrence in the jawakuno collection either.
In app4840. J1 should be listed only once. 
Can app4830 and app4831 be merged into a single apparatus entry?

- AK.02.09.021 	kiṃśāruḥ sasyaśūkaṃ syāt kaṇiśaṃ sasyamañjarī |
				dhānyaṃ vrīhiḥ stambakariḥ stambo gucchastṛṇādinaḥ ||
- AbhCM 		dhānyaṃ tu sasyaṃ sītyaṃ ca vrīhiḥ stambakariś ca tat |
- Kośakalpataru 	kāṣṭhe pañcāṅgulaṃ klībe dhānyaṃ sītyaṃ ca sasyakam |
				vrīhiḥ stambakarī puṃsi vrīhistvāśuśca pāṭalaḥ || 28 ||

332 Synonyms of of Hibiscus Flower
	1 oḍrapuṣpa, japakusuma, ṅa vuṅa vari. 
	^4828. oḍrapuṣpa] em., oṇḍrapuṣpa EdLC, moprapuspa J1 J2, lac. J3 (larger gap), lac. J1 (larger gap)
	^4829. vuṅa vari] J1 J2, vuṅasāri EdLC, lac. J3 (larger gap), lac. J1 (larger gap)
	
Undo repetition of "of" in the heading.
I think you should edit japākusuma. See the parallels.
The reading mopra for oḍra us anbother cases of p/ḍ errors. We have seen several before.
I don't understand why your apparatus gives readings for J1 if this segment falls within in a larger gap.

-   AK.02.04.076ab uḍupuṣpaṃ japā puṣpaṃ vajrapuṣpaṃ tilasya yat |
    AK.02.04.076cd pratihāsaśataprāsacaṇḍātahayamārakāḥ || 76 ||
    AK.02.04.077ab karavīre karīre tu krakaragranthilāvubhau .
- `<Amarapadavivrti>` uḍupuṣpam iti — uḍudeśe bhavaṃ puṣpaṃ uḍupuṣpam | oḍrapuṣpam iti vā pāṭhaḥ | ...
- AṬS *bhadrapuṣpaṃ japā* 'oḍrapuṣpaṃ japā' ity eva pāṭhaṃ bhānujidīkṣito 'py upādatte |

AbhRM 207c	oḍrapuṣpam abhidhīyate japā,
AbhRM 738cd japākusumasaṃkāśā lohinī parikīrtitā || 738 ||

331 Synonyms of Safflower
	1 mahārajana, ⟨ku⟩sumbha, ṅa kasumbha. 
	^4826. ⟨ku⟩sumbha] conj. EdLC, śamba J1 J2, lac. J3 (larger gap), lac. J1 (larger gap)
	^4827. kasumbha] J2, yumba EdLC J1, lac. J3 (larger gap), lac. J1 (larger gap) • See 17.26.

I don't understand why your apparatus gives readings for J1 if this segment falls within in a larger gap.

You wrote: "AK 3.3.633: syān mahārajate klībaṁ kusumbhaṁ karake pumān |. It is remarkable that all AK editions show mahārajata but the other kośa texts have mahārajana." This is now proved to be wrong by the Adyar Libary edition (ed. Ramanathan). The commentary Amarapadavivarana also dives no sign of awareness of a reading with t:
	
- AK.03.03.136cd syān mahārajane klībaṃ kusumbhaṃ karake pumān || 136 ||
- `<Amarapadavivarana>` ... syān mahārajane—pumān | mahārajanaṃ vahniśikham (1, pṛ. 616) | padmakam iti svāmī (pṛ. 302) | ... `</Amarapadavivarana>`

So I think you must adopt the reading mahārajane and change accompanying sentence: "Some editions reading `<foreign>`mahārajate</foreign> for `<foreign>`mahārajane`</foreign>`."
I am not sure why you have highlighted kosumbha in the AṬS parallel. It casts no light on the existence of the spelling kasumbha in the Old Javanese tradition.

You have wrongly cited the AbhRM parallel, which actually read like this:

AbhRM	mahārajanam icchanti kusumbhaṃ ca sumedhasaḥ || 620 ||

330	Synonyms of Moss
	1	śaivala, ṅa lumut. 

This is another case where use of only one Skt. term makes it impossible to say anything about particular proximity with the one or the other Skt. kośa.

- AK.01.12.039ab jalanīlī tu śaivālaṃ śaivalo 'tha kumudvatī |
  AK.01.12.039cd kumudinyāṃ nalinyāṃ tu bisinīpadminīmukhāḥ || 39 ||
- AbhCM	utpalānāṃ tu śālūkaṃ nīlyāṃ śaivālaśevale |
		śevālaṃ śaivalaṃ śepālaṃ jalācchūkanīlike || 1167 ||
- AbhRM	śevālaṃ śaivalaṃ proktaṃ jalaśūkaṃ ca nīlikā || 683 ||

329 Synonyms of Bud
	1 mukula, kuḍmala, ṅa kumucup. 
	^4824. mukula] em. EdLC, kumula J1 J2, lac. J3 (larger gap), lac. J1 (larger gap)
	^4825. kuḍmala] EdLC J1, kumla J2, lac. J3 (larger gap), lac. J1 (larger gap)

It is interesting that the OJ give a very dynamic form (k-um-ucup) so I suggest Remaining in Bud, as opposed to Blossoming in 328. 
I don't understand why your apparatus gives readings for J1 if this segment falls within in a larger gap.

I find two occurrences of kumula (as in your mss.) instead if mukula. I cannot exclude they are typos in the e-texts, so you should look uop the printed editions:

- Nānārthārṇasaṁkṣepa
kumulaṃ kusume klībaṃ hiraṇye ca dvayoḥ punaḥ || 538 ||
- Vāṅmayārṇava
kumulaṃ puṣpahemnoḥ klī kānte triṣu śiśau dvayoḥ || 1456 ||

All usual parallels give mukula, so for the time being I still think EdLC's em. should be accepted.

- AK.02.04.016ab kṣārako jālakaṃ klībe kalikā korakaḥ pumān |
   AK.02.04.016cd syād gucchakas tu stabakaḥ kuḍmalo mukulo 'striyām || 16 ||
- AbhCM	kuḍmale mukulaṃ guñche gucchas tabakagutsakāḥ |
		guluñcho 'tha rajaḥ pauṣpaṃ parāgo 'tha raso madhu || 1126 ||
- AbhRM	puṣpaṃ prasavaḥ kusumaṃ prasūnakaṃ sumanasaḥ samākhyātāḥ |
		korakajālakakalikākuḍmalamukulāni tulyāni || 186 ||

328	Synonyms of Blossoming
	1 aśoka, puṣita, utpala, vikasat, ṅa sumkar. 
	
I think we expect something like vyākośa for the first term; puṣpita (have you forgotten a p?) for the second; and utphulla for the third. See the parallels.

Normalize sumkar as suməkar.
	
- AK.02.04.007ab vandhyo 'phalo 'vakeśī ca phalavān phalinaḥ phalī |
  AK.02.04.007cd praphullotphullasaṃphullavyākośavikacasphuṭāḥ || 7 ||
  AK.02.04.008ab phullaścaite vikasite syuravandhyādayastriṣu |
- AbhCM prabuddhojjṛmbhaphullāni vyākośaṃ vikacaṃ smitam || 1127 ||
		unmiṣitaṃ vikasitaṃ dalitaṃ sphuṭitaṃ sphuṭam |
		praphullotphullasaṃphullocchvasitāni vijṛmbhitas || 1128 ||
- AbhRM	udbuddhaṃ vyākośaṃ puṣpeṣu vikāśavācakāḥ śabdāḥ || 187 ||

327

Synonyms of Falsehood
	1 anr̥ta, vitatha, alīka, abhūta, ṅa lñok. 
	^4818. vitatha, alīka] em., lac. EdLC, vika[... J1, vikaphalīka J2, lac. J3 (larger gap), lac. EdLC (larger gap)
	^4820. abhūta] J2, vitatha EdLC, lac. J3 (larger gap), lac. EdLC (larger gap), lac. J1 (larger gap)

EdLC appear twice in app4818 and app4820.
You have forgotten the | at the end of the second parallel.
I think the text must be edited something like this: 

	anr̥ta, viphala, abhūta

But the parallels don't seem to help to make a confident decision. 
Normalize lñok as ləñok.

- AK.03.04.015ab mṛṣā mithyā ca vitathe yathārthaṃ tu yathātatham |
  AK.03.04.015cd syur evaṃ tu punar vai vety avadhāraṇavācakāḥ || 15 ||

the AbhCM parallel should rather be shown like this:

satyaṃ samyak samīcīnam ṛtaṃ tathyaṃ yathātatham || 264 ||
yathāsthitaṃ ca sadbhūte'līke tu vitathānṛte |
	yathāsthitaṃ is a single word!!!
atha kliṣṭaṃ saṃkulaṃ ca parasparaparāhatam || 265 ||

- Kośakalpataru
kṣiproktaṃ tu nirastam āhatam idaṃ mithyārthakaṃ syādathā-
vadhya(dyaṃ?)vyartha[46a]mabaddhakaṃ tu viphalaṃ coccāvacaṃ tadbhavet || 52 ||
	maybe to be corrected as follows (without thinking about meter yet):
kṣiproktaṃ tu nirastam āhatam idaṃ mithyārthakaṃ syād a[ya]thāvad vyartham abaddhakaṃ tu viphalaṃ coccāvacaṃ tad bhavet || 52 ||

- Nāmamālā
mṛṣālīkaṃ mudhā moghaṃ viphalaṃ vitathaṃ vṛthā |
vidhuraṃ vyasanaṃ kaṣṭaṃ kṛcchraṃ gahanam uddharet || 186 ||

326 Synonyms of Covering
	1 ācchādhana, ṅa tavəṅ-tavəṅ. 
	
The Skt. term should be ācchādana.

- AK.03.03.125ab ācchādanaṃ saṃpidhānam apavāraṇam ity ubhe |
...
- AK.02.02.015ab gopānasī tu valabhī chādane 'vakradāruṇi |
- AbhCM 	gopāsanī tu valabhīcchādane vakradāruṇi |
		gṛhāvagrahaṇī dehalyumbarodumbaromburāḥ || 1009 ||
- AbhRM 302 ācchādanaṃ syād valabhī gṛhāṇāṃ,
